src/main/java/net/sf/jabref/gui/dbproperties/SaveActionsListModel.java

```diff
@@ -27,20 +27,21 @@ public void addSaveAction(FieldFormatterCleanup action) {
         if (!saveActions.contains(action)) {
             saveActions.add(action);
             for (ListDataListener listener : listeners) {
-                listener.intervalAdded(new ListDataEvent(action, ListDataEvent.INTERVAL_ADDED, saveActions.size(), saveActions.size()));
+                listener.intervalAdded(new ListDataEvent(action, ListDataEvent.INTERVAL_ADDED, saveActions.size(),
+                        saveActions.size()));
             }
         }
     }
 
     public void removeAtIndex(int index) {
-        if (index < 0 || index > saveActions.size()) {
-            throw new IndexOutOfBoundsException("Index must be within 0 and " + saveActions.size() + " but was " + index);
-        }
 
-        FieldFormatterCleanup action = saveActions.remove(index);
+        if ((index >= 0) && (index < saveActions.size())) {
```

Comment:
Shouldn't the error message just be logged or is it often the case that the index parameter is given "out of bounds"?